public/helpers/update-users.js

```diff
@@ -0,0 +1,34 @@
+const updateUsers = (sender, users) => {
+  users.forEach((user) => {
+    if(user.permissions.length == 2){
+      if(mentorExist()){
+        addUser('moderator', user);
+      }
+      else{
+        addUser('mentor', user);
+      }
+    }
+    else{
+      addUser('students', user);
+    }
+  });
+};
+
+const mentorExist = () => {
+  return document.querySelector(".online__mentor-list").hasChildNodes();
+};
+
+const addUser = (type, user) => {
+  let className = createClassName(type);
+  const newUser = document.createElement("LI");
+  newUser.innerHTML = user.username;
+  newUser.classList.add("online__user");
+  let list = document.querySelector(className);
+  list.appendChild(newUser);
+};
+
+const createClassName = (type) =>{
+  const className = ".class__-list";
```

Comment:
Needs to be `const className = ".online__-list";`

So that it will return the correct class; e.g. 'online__students-list'